cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinder.java

```diff
@@ -230,17 +230,17 @@ private void collectLogFlushTimeMetric(BrokerEntity broker,
   }
 
   private void collectPerByteLogFlushTimeMetric(BrokerEntity broker,
-                                                Map<BrokerEntity, ValuesAndExtrapolations> metricsHistoryByBroker,
-                                                Map<BrokerEntity, ValuesAndExtrapolations> currentMetricsByBroker,
+                                                ValuesAndExtrapolations metricsHistory,
+                                                ValuesAndExtrapolations currentMetrics,
                                                 Map<BrokerEntity, List<Double>> historicalPerByteLogFlushTimeMetricValues,
                                                 Map<BrokerEntity, Double> currentPerByteLogFlushTimeMetricValues) {
-    AggregatedMetricValues aggregatedMetricValues = currentMetricsByBroker.get(broker).metricValues();
+    AggregatedMetricValues aggregatedMetricValues = currentMetrics.metricValues();
     double latestLogFlushTime = aggregatedMetricValues.valuesFor(BROKER_LOG_FLUSH_TIME_MS_999TH_ID).latest();
     double latestTotalBytesIn = aggregatedMetricValues.valuesFor(LEADER_BYTES_IN_ID).latest() +
```

Comment:
Question: Is it possible for this `latestTotalBytesIn` variable to be zero? If it's zero,  `latestLogFlushTime / latestTotalBytesIn` throws an exception

Reply:
No -- there is an earlier sanity check to ensure that it is a positive value.